### DISCUSSION TITLE
How to get copter fuel?

### DISCUSSION THREAD

**Discussion by toupzvito** [cataclysmbn/Cataclysm-BN#3437] [Q&A] [answered]
Greetings im having a blast as a copter pilot start... But i have no idea how to get the fuel for the starting copter help please also bright nights is best!

**Comment by chaosvolt** [comment, marked as answer]
Most copters use gas turbines by default, which means they can rely on gasoline just fine as well as avgas and jp8. Main issue if you spawn with your tank of the harder-to-find stuff will likely be that you'd have to decide whether to dump your fuel to switch to gas when it gets low, weld a spare tank on to fill with gasoline, or keep pushing for a source of the good stuff, since can't mix fuels in the tank.

As for where to find avgas and jp8, main sources are airports and especially military helipads. The former is pretty much a freebie aside from some zombies to fend off, while the latter has turrets to deal with too.

**Reply by toupzvito** [reply]
Thank you so much :D
